hack/windows/Clean-AntreaNetwork.ps1

```diff
@@ -103,15 +113,34 @@ function ClearHnsNetwork() {
 
 clearOVSBridge
 ClearHnsNetwork
-if ($RemoveOVS) {
-    RemoveOVSService
-} else {
-    # ResetOVSService is called to recover Windows Services "ovsdb-server" and "ovs-vswitchd" if they are removed
-    # unexpectedly or their status is not correct, e.g., ovs-vswitchd fails to go into Running.
-    # This might happen after the Windows host is restarted abnormally, in which case some stale configurations
-    # can prevent ovs-vswitchd from running, like a stale pid file or misconfigurations in OVSDB.
-    ResetOVSService
+switch ($OVSRunMode) 
+{
+    "Service" {
+        if ($RemoveOVS) {
+            RemoveOVSService
+        } else {
+            # ResetOVSService is called to recover Windows Services "ovsdb-server" and "ovs-vswitchd" if they are removed
+            # unexpectedly or their status is not correct, e.g., ovs-vswitchd fails to go into Running.
+            # This might happen after the Windows host is restarted abnormally, in which case some stale configurations
+            # can prevent ovs-vswitchd from running, like a stale pid file or misconfigurations in OVSDB.
+            ResetOVSService
+        }
+    }
+    "Pod" {
+        if (Test-Path -Path $OVS_DB_PATH) {
+            Remove-Item -Path $OVS_DB_PATH -Force
+            if (Test-Path -Path $OVS_DB_PATH) {
+                Write-Host "Failed to delete file: $OVS_DB_PATH"
+                exit 1
+            }
+        }
+    }
+    default {
```

Comment:
If the default value of `OVSRunMode` is "Service", we should allow user running this script without `OVSRunMode`